vscode/src/client/analyzerClient.ts

```diff
@@ -18,30 +18,26 @@ import {
   SolutionState,
   Violation,
 } from "@editor-extensions/shared";
-import { paths, fsPaths, ignoresToExcludedPaths } from "../paths";
```

Comment:
Are we losing this capability? Maybe I missed it but I do not see anything in starting the analyzer or running the analysis that handles this.

Reply:
We are not, right now, analysis isn't working. I have this under outstanding work, but thanks for pointing out, I definitely would have missed this detail.